<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormReviews</class>
 <widget class="QWidget" name="FormReviews">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отзывы о товарах</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/32/icons/32x32/add.png</normaloff>:/32/icons/32x32/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonEdit">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/32/icons/32x32/edit.png</normaloff>:/32/icons/32x32/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonDelete">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/32/icons/32x32/delete.png</normaloff>:/32/icons/32x32/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRefresh">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/32/icons/32x32/refresh.png</normaloff>:/32/icons/32x32/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FrameSearchTable" name="widgetSearch" native="true"/>
   </item>
   <item>
    <widget class="QTableView" name="tableViewReviews">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="FramePageNavigator" name="widgetNavigator" native="true"/>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/add.png</normaloff>:/32/icons/32x32/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить отзыв</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/edit.png</normaloff>:/32/icons/32x32/edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="toolTip">
    <string>Редактировать отзыв</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/delete.png</normaloff>:/32/icons/32x32/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить отзыв</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/refresh.png</normaloff>:/32/icons/32x32/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить данные</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameSearchTable</class>
   <extends>QWidget</extends>
   <header>framesearchtable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FramePageNavigator</class>
   <extends>QWidget</extends>
   <header>framepagenavigator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
